client/src/components/header.tsx

```diff
@@ -197,35 +198,34 @@ function Header(): JSX.Element {
       data-cy="header"
       data-mentor={mentor._id}
       style={{
-        display: "flex",
-        flexDirection: "column",
-        justifyContent: "center",
-        height: 55,
         backgroundColor: `${styleHeaderColor}`,
         color: `${styleHeaderTextColor}`,
       }}
+      className="header-container"
     >
-      {mentor.name !== "USC" ? (
-        <IconButton
-          aria-label="information"
-          component="span"
-          style={{
-            position: "absolute",
-            left: "50px",
-            color: `${styleHeaderTextColor}`,
-          }}
-          onClick={handleClickHome}
-          data-cy="home-button"
-        >
-          <HomeIcon />
-        </IconButton>
-      ) : (
-        ""
-      )}
-
-      <Typography>
-        {mentor.name}: {mentor.title}
-      </Typography>
+      <div className="home-btn-wrapper">
+        {mentor.name !== "USC" ? (
```

Comment:
This seems really specific to have for a single mentor case, what was the purpose of this check again? Is it something we can move to config or a user setting?

Reply:
I'm not sure, it was already there before my changes. I don't know if we can move it to config.